packages/flutter/lib/src/painting/utils.dart

```diff
@@ -0,0 +1,55 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math' as math;
+
+import 'package:flutter/foundation.dart';
+
+import 'basic_types.dart';
+
+const double _kScreenEdgeMargin = 10.0;
+
+/// Position a box either above or bellow a target box specified in the global
+/// coordinate system.
+///
+/// The target box is specified by [size] and [target] and the box being
+/// positioned is specified by [childSize]. [verticalOffset] is the amount of
+/// vertical distance between the boxes.
+///
+/// Used by [Tooltip] to position a tooltip relative to its parent.
+///
+/// The arguments cannot be null.
```

Comment:
Fixed here and sent out https://github.com/flutter/flutter/pull/11683
so that the next person documenting non-nullable arguments isn't tripped up by
existing examples of "cannot be null" that should have been "must not be null".